### PR TITLE
[14.0][IMP] shopinvader.partner form -- use sheet

### PR DESCRIPTION
**Pull request by ivantodorovich**
**Before**:
image

**After**:
image

### CODE REVIEW

**Comment by ivantodorovich** [comment]
pre-commit is failing for some incompatiblity with `python3.10`, not really related to this PR

**Review by SilvioC2C — APPROVED** [review]

File: shopinvader_sale_profile/views/shopinvader_partner.xml [hunk 1]
```diff
@@ -3,10 +3,18 @@
      Copyright 2018 ACSONE SA/NV
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo>
+
+    <record model="ir.ui.view" id="shopinvader_partner_view_form">
+        <field name="model">shopinvader.partner</field>
+        <field name="inherit_id" ref="shopinvader.shopinvader_partner_view_form" />
+        <field name="arch" type="xml">
+            <field name="partner_email" position="after">
+                <field name="sale_profile_id" widget="selection" />
```
Comment (simahawk) [inline shopinvader_sale_profile/views/shopinvader_partner.xml:12]: I'm not sure this is nice when you have 20-30 profiles...

**Review by simahawk — APPROVED** [review]

**Comment by ivantodorovich** [comment]
Merge failed because of https://github.com/shopinvader/odoo-shopinvader/pull/1160 btw.
We should merge that first

**Comment by ivantodorovich** [comment]
> /ocabot merge minor

@simahawk could you retry ?

**Pull request merged**